src/main/resources/metafile-schema.json

```diff
@@ -114,6 +114,7 @@
           "document_type": {
             "id": "#/properties/scannable_items/items/properties/document_type",
             "type": "string",
+            "pattern": "^([Cc][Hh][Ee][Rr][Ii][Ss][Hh][Ee][Dd]|[Oo][Tt][Hh][Ee][Rr])$",
```

Comment:
How about picking uppercase or lowercase and enforcing that one?

Reply:
I think the specs agree on `first capital` only. Kept it 100% case insensitive as those words form a "short" regex

Reply:
completely forgot about flags. and i was googling something about it before starting to work. will check this tomorrow

Reply:
so tried many different ways and looks like json schema does not support regex flags. nor does support case insensitive enums. this is "the most elegant" way

only two words is not that bad. what we could do is completely ignore this and just trust object mapper implementation of mapping to java enum

Reply:
I think `(?i)` works in Java. But we should only use it if the spec wants us to be case insensitive. Otherwise we shouldn't be more liberal than the spec is.

Reply:
Aaa, fair enough. Adding the restricting to schema was a good thing, though. How about just `$(Cherished|Other)` or something?

Reply:
Then it is just an enum really. And it will receive same treatment as `classification`. Ideally don't want to bound to case sensitivity especially when [this feature is enabled](https://github.com/hmcts/bulk-scan-processor/blob/demo/src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/MetafileJsonValidator.java#L19). Makes json schema validation irrelevant. Plus it is not that smart to allow some more complex rules